components/toast/index.css

```diff
@@ -151,21 +128,16 @@
 	}
 }
 
-.spectrum-Toast-buttons {
-	display: flex;
-	flex: 0 0 auto;
-	align-items: flex-start;
+.spectrum-CloseButton {
```

Comment:
We'll probably have to scope this selector to the toast. Either by nesting this within `.spectrum-Toast` (similar to before), or maybe giving this a custom class (like `.spectrum-Toast-closeButton`)